db/seeds.rb

```diff
@@ -39,63 +39,64 @@
   r.time_zone = 'America/Los_Angeles'
 end.save!
 
-primary_location = Location.find_by(city_name: ENV['PRIMARY_CITY_NAME'])
-Widget.find_or_initialize_by(
-  name: "Guests"
-) do |r|
-  r.enabled = false
-  r.duration_seconds = 20
-  r.position = 0
-  r.sidebar_text = 'Guests'
-  r.show_weather = false
-  r.location = primary_location
-end.save!
+Location.all.each do |location|
```

Comment:
Maybe we only do this for `Providence`, `Boulder`, and `San Diego`?

The other gotcha here is we can't `db:seed` the production database since it data already like events. We could use a migration to populate but that also doesn't feel right. I'm going to mull this over for a bit. I'm open to suggestions.

Reply:
It seems like seeds.rb is the only way to maintain the production database until we have an admin interface. I do think we can skip NYC and DC here since they're artifacts from an old design.

Reply:
If they are invalid we should drop them from the database. That doesn't need to happen here though, keeping this general case makes sense.